app/controllers/api/v1/task_category_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::TaskCategoryController < ApplicationController
+  before_action :require_admin, only: [:index, :delete, :update, :create]
```

Comment:
normal users should be able to hit `index` api, else how'll they know which categories are available?